internal/media/store.go

```diff
@@ -90,12 +91,12 @@ const (
 func (store *Store) SaveMovie(db *sqlx.DB, movie *Movie) error {
 	var updatedMovie Movie
 	if err := db.QueryRowx(`
-		INSERT INTO movie(id, tmdb_id, title, adult, source_path, created_at, updated_at)
-		VALUES($1, $2, $3, $4, $5, current_timestamp, current_timestamp)
+		INSERT INTO media(id, type, tmdb_id, title, adult, source_path, created_at, updated_at)
+		VALUES($1, $2, $3, $4, $5, $6, current_timestamp, current_timestamp)
 		ON CONFLICT(tmdb_id) DO UPDATE
-			SET (updated_at, title, adult, source_path) = (current_timestamp, EXCLUDED.title, EXCLUDED.adult, EXCLUDED.source_path)
-		RETURNING *
-	`, movie.ID, movie.TmdbId, movie.Title, movie.Adult, movie.SourcePath).StructScan(&updatedMovie); err != nil {
+			SET (updated_at, title, adult, source_path, type) = (current_timestamp, EXCLUDED.title, EXCLUDED.adult, EXCLUDED.source_path, 'movie')
```

Comment:
Hm, just realised this is a problem. Using only the tmdb_id to detect conflicts sounds like a good idea, but today while messing with their API I discovered their IDs are only unique in their category (i.e. movies and series with the same ID do exist). We should probably make this constraint more robust by only considering a row to be conflicting if it has the same ID **and** type (movie|episode).

Reply:
Should be simple to just modify the constraint for this PR right?

Reply:
Yeah fair enough, I'll update the initial schema UK and the relevant query now

Reply:
Okay, updated this to look for conflicts over both `tmdb_id` and `type`. Also replaced the simple column `UNIQUE` contraint with a composite unique contraint.

Reply:
New commit looks good!